### ISSUE TITLE
Accessibility of search filter form and behavior

### ISSUE THREAD

**Issue by admturner** [washingtonstateuniversity/hrswp-plugin-blocks#2]
**Describe the solution you'd like**

The search form should be accessible -- with at least a label connecting the search input field with the content to be searched, and probably some announcement of the content being changed.

**Comment by admturner** [comment]
The "aria-live" attribute seemed inappropriate for this feature during testing. It announced too much to be particularly useful, especially for long sections of searchable content. Nevertheless, we've set the attribute on the searchable content container to indicated that it is a live region, but set the value to "off" to prevent noise.
